jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -160,8 +160,10 @@ MlirToJlmConverter::ConvertCmpIOp(
 rvsdg::node *
 MlirToJlmConverter::ConvertBitBinaryNode(
     ::mlir::Operation & mlirOperation,
-    std::vector<rvsdg::output *> & inputs)
+    ::llvm::SmallVector<rvsdg::output *> & inputs)
 {
+  if (inputs.size() != 2)
+    return nullptr;
```

Comment:
Why is that here? How can it be that a binary node has more than two operands?

Also, just skipping the conversion for a node is not really an option, is it? It also means its results are not defined for the next operation etc. This would ultimately mean that the entire conversion is failing, no?